api/src/pcapi/core/offers/api.py

```diff
@@ -1960,8 +1960,8 @@ def get_stocks_stats(offer_id: int) -> StocksStats:
         )
 
 
-def check_can_move_event_offer(offer: models.Offer) -> list[offerers_models.Venue]:
-    if not offer.isEvent:
+def check_can_move_offer(offer: models.Offer, only_event: bool = True) -> list[offerers_models.Venue]:
```

Comment:
[Low LOGAF] peut être qu'on pourrait avoir un nom plus explicite (p.ex du FF comme ça en plus on voit que c'est lié)
```suggestion
def check_can_move_offer(offer: models.Offer, only_move_event_offer: bool = True) -> list[offerers_models.Venue]:
```